content/en/docs/tasks/access-application-cluster/service-access-application-cluster.md

```diff
@@ -46,77 +46,81 @@ provides load balancing for an application that has two running instances.
     each of which runs the Hello World application.
 
 1. Display information about the Deployment:
-
-       kubectl get deployments hello-world
-       kubectl describe deployments hello-world
+   ```
+   kubectl get deployments hello-world
+   kubectl describe deployments hello-world
+   ```
 
 1. Display information about your ReplicaSet objects:
-
-       kubectl get replicasets
-       kubectl describe replicasets
+   ```
+   kubectl get replicasets
+   kubectl describe replicasets
+   ```
 
 1. Create a Service object that exposes the deployment:
-
-       kubectl expose deployment hello-world --type=NodePort --name=example-service
+   ```
+   kubectl expose deployment hello-world --type=NodePort --name=example-service
+   ```
 
 1. Display information about the Service:
-
-       kubectl describe services example-service
-
-    The output is similar to this:
-
-        Name:                   example-service
-        Namespace:              default
-        Labels:                 run=load-balancer-example
-        Annotations:            <none>
-        Selector:               run=load-balancer-example
-        Type:                   NodePort
-        IP:                     10.32.0.16
-        Port:                   <unset> 8080/TCP
-        TargetPort:             8080/TCP
-        NodePort:               <unset> 31496/TCP
-        Endpoints:              10.200.1.4:8080,10.200.2.5:8080
-        Session Affinity:       None
-        Events:                 <none>
-
-    Make a note of the NodePort value for the service. For example,
-    in the preceding output, the NodePort value is 31496.
+   ```
+   kubectl describe services example-service
+   ```
```

Comment:
I think it's convention (was required in Jekyll) to have a blank line between two consecutive code fences or a code fence and another block-level element like a paragraph..

Reply:
You might be referring to something more specific, but in general leaving a blank line between a code fence and text in this ordered list caused the list to start renumbering.  For example, adding a blank line between `1. Display information about the Service:` and the code fence that follows rendered like this when I tested locally (with hugo .40.3)


![2018-05-25_10-38-44](https://user-images.githubusercontent.com/216054/40558460-3867a2ac-6008-11e8-8993-7afcf361c551.png)

Reply:
This is one of this situations where adding an extra level of indent to the codeblock works, but maybe if Hugo doesn't mind not having the blank lines, that's cleaner.